test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file should ideally be ignored by git when you push your changes. The purpose of the `.gitignore` file is to ignore all files added to it, so it's not added to your repository. Kindly check your files changed, and make sure it ignored on this branch 👍 

## Instructions:
![image](https://github.com/jovasal/My-First-Project/assets/50990925/8a6886fa-c43e-4f2c-8016-37e665cf805b)

Reply:
I delete "This file should be ignored by git"
![image](https://github.com/jovasal/My-First-Project/assets/35693330/047ea945-a34a-4700-a777-f79a954aa2d0)

Also, Test.md is into .gitignore

![image](https://github.com/jovasal/My-First-Project/assets/35693330/bd3c8f64-3683-43b3-a1f7-00771adf195f)